### DISCUSSION TITLE
PostgreSQL versions support for Prisma 1

### DISCUSSION THREAD

**Discussion by emmenko** [prisma/prisma#9582] [Q&A] [answered]
Hi folks 👋,

Does anyone know which PostgreSQL version is supported by Prisma 1?

**Background**

We're still running on Prisma `v1.34` using AWS Aurora PostgreSQL `9.6`. However, the Aurora database will stop supporting version `9.6` by the end of 01/2022, so we need to upgrade the database to a newer version.

In the Prisma docs I could only find the [supported databases](https://www.prisma.io/docs/reference/database-reference/supported-databases) for Prisma 2, but I'm not sure if that list also applies to Prisma 1.
Could someone clarify what PostgreSQL versions are "officially" supported by Prisma 1?

For instance, we do plan to migrate to Prisma 2 next year, so knowing the supported DB versions for both Prisma 1 and 2 would be great. For example, if v13 is only supported by Prisma 2, it probably does not make sense to upgrade to v13 now.

Thanks a lot for any help.

**Comment by ryands17** [comment, marked as answer]
@emmenko 👋 

We used to test till Postgres version 11, so it should work fine till then.

**Reply by emmenko** [reply]
Thanks for the quick answer! That should give us enough information for now.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
